src/client/analyticsBeaconClient.ts

```diff
@@ -26,11 +26,14 @@ export class AnalyticsBeaconClient implements AnalyticsRequestClient {
         };
 
         // tslint:disable-next-line: no-console
-        console.log(`Sending beacon for "${eventType}" with: `, JSON.stringify(payload));
         navigator.sendBeacon(url, body as any); // https://github.com/microsoft/TypeScript/issues/38715
         return;
     }
 
+    public isAvailable() {
+        return typeof navigator.sendBeacon === 'function';
```

Comment:
It's funny how I still have a trigger for this, because of Internet Explorer (shows my age) that would report `object` instead of `function` for built-in methods on the web API.

That aside, I do wonder why you use this stricter check over the `in` operator? It's what MDN suggest for [Implementing feature detection](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#javascript).
```suggestion
        return 'sendBeacon' in navigator;
```